src/unix/mod.rs

```diff
@@ -447,6 +458,11 @@ extern {
                link_name = "waitpid$UNIX2003")]
     pub fn waitpid(pid: pid_t, status: *mut ::c_int, options: ::c_int)
                    -> pid_t;
+    #[cfg(not(target_os = "openbsd"))] // " if " -- appease style checker
```

Comment:
Could the definition here be pushed down into the various modules to avoid the `#[cfg]`?